doc/library/config.txt

```diff
@@ -416,32 +416,36 @@ import theano and print the config variable, as in:
     `amdlibm <http://developer.amd.com/cpu/libraries/libm/>`__
     library, which is faster than the standard libm.
 
-.. attribute:: config.lib.cnmem
+.. attribute:: config.gpuarray.preallocate
 
-    Float value: >= 0
+    Float value
 
-    Controls the use of `CNMeM <https://github.com/NVIDIA/cnmem>`_ (a
-    faster CUDA memory allocator). In Theano dev version until 0.8
-    is released.
+    Default: 0 (Preallocation of size 0, only cache the allocation)
```

Comment:
@nouiz, @chinnadhurai did some change here but merged it with the previous commit.